sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/OtlpConfigUtil.java

```diff
@@ -171,6 +171,30 @@ private static URL validateEndpoint(@Nullable String endpoint, boolean allowPath
     return endpointUrl;
   }
 
+  private static byte[] readPemByConfig(ConfigProperties config, String prefix, String dataType, String suffix ) {
+    String propertyToRead = prefix + "." + dataType + "." + suffix;
+    String filePath = config.getString(propertyToRead);
+    if (filePath == null) {
+      propertyToRead = prefix + "." + suffix;
+      filePath = config.getString(propertyToRead);
+    }
+    if (filePath != null) {
+      Path path = Paths.get(filePath);
+      if (!Files.exists(path)) {
+        throw new ConfigurationException("Invalid file: " + path + " (configured in property: " + propertyToRead +")");
+      }
+      byte[] fileBytes;
+      try {
+        fileBytes = Files.readAllBytes(path);
```

Comment:
Can just return here and save a few lines:
```
return Files.readAllBytes(path)
```

Reply:
fixed